app/controllers/students_controller.rb

```diff
@@ -1,7 +1,10 @@
 class StudentsController < ApplicationController
   before_action :authenticate_user!, :set_user
 
-  def show; end
+  def show
+    @reviews = Review.where(student_id: @user.id).last(5)
+    @assigned_learning_resouces = StudentsLearningResource.where(user_id: @user.id).last(5)
```

Comment:
I'd expect `StudentsLearningResource` to be linked to a [student](https://github.com/bryanmg/ammdk_colima/blob/6cc660e26103c5ff0787ffbb3f64b26934f47d92/app/models/students_learning_resource.rb#L2), not a user. 
You don't have to chance this, but it's not clear enough